src/gr11review/part1/Review1.java

```diff
@@ -1 +1,61 @@
 package gr11review.part1;
+
+import java.io.*;
+
+public class Review1{
+
+    public static void main(String[] args) throws IOException{
+        BufferedReader month = new BufferedReader(new InputStreamReader(System.in));
+        BufferedReader day = new BufferedReader(new InputStreamReader(System.in));
+
+
+        int intMonth; 
+        int intDay;
+        int intDays;
+
+        System.out.print("Enter the month number: ");
+        intMonth = Integer.parseInt(month.readLine());
+
+        System.out.print("Enter the day number: ");
+        intDay = Integer.parseInt(day.readLine());
+
+        if(intMonth == 1){
+            intDays = intDay;
+            System.out.print(intDays);
```

Comment:
Consider outputting with `println`. This way, when the program finishes execution, you return to the CLI on a new line, not with the command prompt on the same line as the last line of output.